src/data/toki_pona_dictionary.json

```diff
@@ -1,1712 +1 @@
ma", "meanings": [["comp", "tomorrow"]]}, {"word": "tenpo suno ni", "meanings": [["comp", "today"]]}, {"word": "toki", "meanings": [["n", "language, talking, speech, communication"], ["vt", "say"], ["vi", "talk, chat, communicate"], ["interj", "hello! hi!"]]}, {"word": "toki Inli", "meanings": [["comp", "English"]]}, {"word": "toki Lasina", "meanings": [["comp", "Latin"]]}, {"word": "toki suli", "meanings": [["comp", "announcement"]]}, {"word": "tomo", "meanings": [["n", "indoor constructed space, e.g. house, home, room, building"], ["mod", "urban, domestic, household"]]}, {"word": "tomo anpa", "meanings": [["comp", "basement"]]}, {"word": "tomo lape", "meanings": [["comp", "bedroom"]]}, {"word": "tomo lawa", "meanings": [["comp", "headquarters, courtroom"]]}, {"word": "tomo len", "meanings": [["comp", "tent"]]}, {"word": "tomo lipu", "meanings": [["comp", "library"]]}, {"word": "tomo mani", "meanings": [["comp", "bank"]]}, {"word": "tomo moku", "meanings": [["comp", "restaurant"]]}, {"word": "tomo pali", "meanings": [["comp", "studio"]]}, {"word": "tomo sewi", "meanings": [["comp", "temple, church"]]}, {"word": "tomo sona", "meanings": [["comp", "academy"]]}, {"word": "tomo tawa", "meanings": [["comp", "car, vehicle"]]}, {"word": "tomo tawa telo", "meanings": [["comp", "boat"]]}, {"word": "tomo telo", "meanings": [["comp", "bathroom"]]}, {"word": "tonsi", "meanings": [["mod*", "queer, non-binary, gender non-conforming"]]}, {"word": "tu", "meanings": [["mod", "two"], ["n", "duo, pair"], ["vt", "double, separate/cut/divide in two"]]}, {"word": "unpa", "meanings": [["n", "sex, sexuality"], ["mod", "erotic, sexual"], ["vt", "have sex with, sleep with, fuck"], ["vi", "have sex"]]}, {"word": "uta", "meanings": [["n", "mouth"], ["mod", "oral"]]}, {"word": "utala", "meanings": [["n", "conflict, disharmony, competition, fight, war, battle, attack, blow, argument, physical or verbal violence"], ["vt", "hit, strike, attack, compete against"]]}, {"word": "walo", "meanings": [["mod", "white, light (colour)"], ["n", "white thing/part, whiteness, lightness"]]}, {"word": "wan", "meanings": [["mod", "one, a"], ["n", "unit, element, particle, part, piece"], ["vt", "unite, make one"]]}, {"word": "waso", "meanings": [["n", "bird, winged animal"]]}, {"word": "waso telo", "meanings": [["comp", "duck"]]}, {"word": "wawa", "meanings": [["n", "energy, strength, power"], ["mod", "energetic, strong, fierce, intense, sure, confident"], ["vt", "strengthen, energize, empower"]]}, {"word": "wawa ala", "meanings": [["comp", "weak"]]}, {"word": "weka", "meanings": [["mod", "away, absent, missing"], ["n", "absence"], ["vt", "throw away, remove, get rid of"]]}, {"word": "weka sona", "meanings": [["comp", "forget"]]}, {"word": "wile", "meanings": [["vt", "to want, need, wish, have to, must, will, should"], ["n", "desire, need, will"], ["mod", "necessary"]]}, {"word": "wile moku", "meanings": [["comp", "hunger, hungry"]]}, {"word": "wile sona", "meanings": [["comp", "curiosity"]]}]
```

Comment:
Why is this file being minified? This file is not directly served to the user.

Reply:
I used a quick Python script to add the compounds instead of adding them by hand. Python minifies JSON by default.

I could prettyprint it instead, if you prefer?

Reply:
Please. This file is sometimes edited by hand to adjust meanings or add single new ones.

Reply:
JSON is now prettified